### PR TITLE
Add FAQs for JDBC Sink: FAQ PR #4

### PR DESCRIPTION
**Pull request by Meghajit**
This PR does the following:-

builds upon the FAQ section as created in the [FAQ PR # 3](https://github.com/odpf/firehose/pull/139)
adds FAQs for JDBC Sink and references them in the Table of Contents

### CODE REVIEW

**Review by kevinbheda — COMMENTED** [review]
Can we link to hikari github repo here ?

File: docs/reference/faq/index.md [hunk 1]
```diff
@@ -0,0 +1,429 @@
...
+Protobuf message, Firehose will serialise this message into a JSON string. While serialising, it will trim insignificant
+whitespaces, retain the default field names as specified in the message descriptor and use the default values for the
+field if no value is set. For a field of type collection of messages, the individual messages are converted into JSON
+strings and then serialised into a JSON array. The array itself is then serialised into a string. For a field of type
+Timestamp, Firehose will parse it into an Instant type and then serialise it as a string. For fields of all other data
+types in the Kafka message, the field values are simply serialised to string.
+
+#### How are the database connections are formed?
+
+DB connections are created using a Hikari Connection Pool.
```
Comment (kevinbheda) [inline docs/reference/faq/index.md:404]: Can we link to hikari pool repo here ?

File: docs/reference/faq/index.md [hunk 2]
```diff
@@ -0,0 +1,429 @@
...
+Unique keys as per the table schema can be configured by setting the config **SINK_JDBC_UNIQUE_KEYS**. For
+example: `order_number,driver_id`. When trying to push a message from Kafka to the database table, Firehose will
+construct a query which first tries to do an INSERT. If there is a conflict, such as a unique key constraint violation,
+the query tries to do an update of all the non-unique columns of the row with appropriate values from the message. If
+still there is a conflict, the query won't do anything.
+
+#### How data types are handled?
+
+Firehose can insert/update rows into the DB table with columns of data type string only. For a field of type as a nested
+Protobuf message, Firehose will serialise this message into a JSON string. While serialising, it will trim insignificant
```
Comment (kevinbheda) [inline docs/reference/faq/index.md:395]: Can we make it as a sublist ?

1. NestedMessages
2. Collections
3. Timestamp

File: docs/reference/faq/index.md (hunk 1 above)
Comment (Meghajit) [reply docs/reference/faq/index.md]: Done as part of bc0843405a62f4f4550bdccb2563a5ffed137bf7

File: docs/reference/faq/index.md (hunk 2 above)
Comment (Meghajit) [reply docs/reference/faq/index.md]: Done as part of bc0843405a62f4f4550bdccb2563a5ffed137bf7

**Pull request merged**